Formula/r/ruby.rb

```diff
@@ -153,11 +153,11 @@ def post_install
     # Since Gem ships Bundle we want to provide that full/expected installation
     # but to do so we need to handle the case where someone has previously
     # installed bundle manually via `gem install`.
-    rm_f %W[
+    rm(%W[
       #{rubygems_bindir}/bundle
       #{rubygems_bindir}/bundler
-    ]
-    rm_rf Dir[HOMEBREW_PREFIX/"lib/ruby/gems/#{api_version}/gems/bundler-*"]
```

Comment:
calling `brew install ruby` now fails for me in this postinstall step if these files don't exist in `rubygems_bindir`. I think the `-f` to `rm` was allowing the operation to succeed even if the files don't exist, and now they can fail

Reply:
fix in https://github.com/Homebrew/homebrew-core/pull/179332